space_robots/README.md

```diff
@@ -0,0 +1,117 @@
+# Space Robots Demo Docker Image
+
+This folder contains the Dockerfile and scripts required to build the Curiosity Mars rover and Canadarm demos into an image called `space_robots_demo`.
+
+The Dockerfile installs all of the prerequisite system dependencies along with the demos source code, then builds the demo code.
+
+The image uses [osrf/space-ros-moveit2:latest](https://hub.docker.com/r/osrf/space-ros-moveit2/tags) as its base image.
+
+## Building the Demo Image
+
+Use the build script provided to build the docker image.
+
+```
+./build.sh
+```
+
+## Running the image container
+
+Run the following to allow GUI passthrough:
+
+```bash
+xhost +local:docker
+```
+
+Then run:
+
+```bash
+./run.sh
+```
+
+## Running the Demos
+
+### Curiosity Mars rover demo
+
+Launch the demo:
+
+```bash
+ros2 launch curiosity_rover_demo mars_rover.launch.py
```

Comment:
Sorry, I haven't been tracking these demos lately but do we also need to launch the GZ sim separately?

```
ros2 launch curiosity_gazebo curiosity_gazebo.launch.py
```

If I do that the rest of the docs work as expected.